### PR TITLE
fix(zh-cn): correct the description of IsLooselyEqual semantic

### LINKED ISSUE #20957: [zh-cn] reverse descriptive semantics
### MDN URL

https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Operators/Equality

### 与此议题相关的小节或标题？

_No response_

### 哪些信息是错误、无用或不完善的？

"Number to String：使用..."

### 你期望看到什么？

String to Number：使用...

### 你有什么支持的链接、参考或引用？

_No response_

### 你还想补充什么内容？

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `zh-cn/web/javascript/reference/operators/equality`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Operators/Equality
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/web/javascript/reference/operators/equality/index.md
* Last commit: https://github.com/mdn/translated-content/commit/523ca6e8c3eefc50fdbf2209a5e6b6de9ccf4ebc
* Document last modified: 2022-10-01T03:20:40.000Z

</details>

### PR DESCRIPTION
**Pull request by yin1999**
### Description

correct the description of IsLooselyEqual semantic

### Related issues and pull requests

Fixes: #20957

### CODE REVIEW

File: files/zh-cn/web/javascript/equality_comparisons_and_sameness/index.md [hunk 1]
```diff
@@ -63,22 +63,22 @@ switch (NaN) {
...
+   - 布尔值（Boolean）：仅当操作数都为 `true` 或都为 `false` 时返回 `true`。
+   - 大整型（BigInt）：仅当两个操作数的值相同时返回 `true`。
+   - 符号（Symbol）：仅当两个操作数引用相同的符号时返回 `true`。
+2. 如果其中一个操作数为 `null` 或 `undefined`，另一个操作数也必须为 `null` 或 `undefined` 以返回 `true`。否则返回 `false`。
+3. 如果其中一个操作数是对象，另一个是原始值，则[将对象转换为原始值](/zh-CN/docs/Web/JavaScript/Data_structures#原始值强制转换)。
+4. 在这一步，两个操作数都被转换为原始值（字符串、数字、布尔值、符号和大整型中的一个）。剩余的转换将分情况完成。
+   - 如果是相同的类型，使用步骤 1 进行比较。
+   - 如果其中一个操作数是符号而另一个不是，返回 `false`。
+   - 如果其中一个操作数是布尔值而另一个不是，则[将布尔值转换为数字](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)：`true` 转换为 1，`false` 转换为 0。然后再次对两个操作数进行宽松比较。
+   - 数字与字符串：[`Number()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/Number) 构造函数相同的算法将字符串转换为数字。转换失败将导致 `NaN`，这将保证相等比较为 `false`。
```
Comment (Josh-Cena) [inline files/zh-cn/web/javascript/equality_comparisons_and_sameness/index.md:79]:
```suggestion
   - 数字与字符串：[将字符串转换为数字](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)。转换失败将导致 `NaN`，这将保证相等比较为 `false`。
```

File: files/zh-cn/web/javascript/equality_comparisons_and_sameness/index.md [hunk 2]
```diff
@@ -63,22 +63,22 @@ switch (NaN) {
...
+   - 数字与大整型：按数值进行比较。如果数值为 ±∞ 或 `NaN`，返回 `false`。
```
Comment (Josh-Cena) [inline files/zh-cn/web/javascript/equality_comparisons_and_sameness/index.md:80]:
```suggestion
   - 数字与大整型：按数值进行比较。如果数字的值为 ±∞ 或 `NaN`，返回 `false`。
```

File: files/zh-cn/web/javascript/reference/operators/equality/index.md [hunk 3]
```diff
@@ -17,24 +19,24 @@ x == y
...
+4. 在这一步，两个操作数都被转换为原始值（字符串、数字、布尔值、符号和大整型中的一个）。剩余的转换将分情况完成。
    - 如果是相同的类型，使用步骤 1 进行比较。
-   - 如果其中一个操作数是 Symbol 而另一个不是，返回 `false`。
-   - 如果其中一个操作数是布尔型而另一个不是，则将布尔型转换为数字：`true` 转换为 1，`false` 转换为 0。然后再次松散地比较两个操作数。
-   - Number to String：使用与 [`Number()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/Number) 构造函数相同的算法将字符串转换为数字。转换失败将导致 `NaN`，这将保证相等是 `false`。
-   - Number to BigInt：按数值进行比较。如果数值为 ±∞ 或 `NaN`，返回 `false`。
-   - String to BigInt：使用与 [`BigInt()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt/BigInt) 构造函数相同的算法将字符串转换为 BigInt。如果转换失败，返回 `false`。
+   - 如果其中一个操作数是符号而另一个不是，返回 `false`。
+   - 如果其中一个操作数是布尔值而另一个不是，则[将布尔值转换为数字](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)：`true` 转换为 1，`false` 转换为 0。然后再次对两个操作数进行宽松比较。
+   - 数字与字符串：[`Number()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/Number) 构造函数相同的算法将字符串转换为数字。转换失败将导致 `NaN`，这将保证相等比较为 `false`。
```
Comment (Josh-Cena) [inline files/zh-cn/web/javascript/reference/operators/equality/index.md:37]:
```suggestion
   - 数字与字符串：[将字符串转换为数字](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)。转换失败将导致 `NaN`，这将保证相等比较为 `false`。
```

File: files/zh-cn/web/javascript/reference/operators/equality/index.md [hunk 4]
```diff
@@ -17,24 +19,24 @@ x == y
...
+   - 数字与大整型：按数值进行比较。如果数值为 ±∞ 或 `NaN`，返回 `false`。
```
Comment (Josh-Cena) [inline files/zh-cn/web/javascript/reference/operators/equality/index.md:38]:
```suggestion
   - 数字与大整型：按数值进行比较。如果数字的值为 ±∞ 或 `NaN`，返回 `false`。
```

**Comment by Josh-Cena** [comment]
👍

**Comment by yin1999** [comment]
Thank you @Josh-Cena. I'm going to merge this :)

**Pull request merged**